client/components/data/query-billing-data/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingBillingData } from 'state/billing-data/selectors';
+import { requestBillingData } from 'state/billing-data/actions';
+
+class QueryBillingData extends Component {
+	static propTypes = {
+		requestingBillingData: PropTypes.bool,
+		requestBillingData: PropTypes.func
+	};
+
+	componentWillMount() {
```

Comment:
re: https://twitter.com/acdlite/status/818930692109242369 though, we should probably update our QueryComponents / docs soon to avoid confusion.

Even the redux example documentation appears to be using some incorrect lifecycle methods.